nav2_bringup/bringup/README.md

```diff
@@ -62,26 +62,31 @@ To view the robot model in RViz:
 
 ## Launch Navigation2 on a Robot
 
-Pre-requisites:
-* Run SLAM or Cartographer with tele-op to drive the robot and generate a map of an area for testing first. The directions below assume this has already been done. If not, it can be done in ROS1 before beginning to install our code.
+### Pre-requisites:
+* Run SLAM with Navigation 2 or tele-op to drive the robot and generate a map of an area for testing first. The directions below assume this has already been done or there is already a map of the area.
+
+    Learn more about how to use Navigation 2 with SLAM to create maps; 
+    - [Navigation 2 with SLAM](https://github.com/ros-planning/navigation2/blob/master/doc/use_cases/navigation_with_slam.md)
+
+
 * Publish all the transforms from your robot from base_link to base_scan
 
-Launch the code using this launch file and your map.yaml:
+### Terminal 1 : Launch Navigation 2 using your map.yaml
```

Comment:
I think the entire subsection titled `Launch Navigation2 on a Robot` can go away. Its just a repeat of the simulation instructions without the simulation artifacts, that should be pretty clear

Reply:
In my opinion, the most important difference is the `use_sim_time:=True` parameter. I think there is a benefit of keeping these sections separate since some users can make a mistake if they don't know what use_sim_time parameter does.

Reply:
Sure, I dont think it hurts.